patroni/ha.py

```diff
@@ -326,21 +326,23 @@ def get_effective_tags(self) -> Dict[str, Any]:
             tags['nosync'] = True
         return tags
 
-    def notify_citus_coordinator(self, event: str) -> None:
-        if self.state_handler.citus_handler.is_worker():
-            coordinator = self.dcs.get_citus_coordinator()
+    def notify_mpp_coordinator(self, event: str) -> None:
```

Comment:
What about adding a docstring if you are touching this code? :)

Reply:
Wording help needed here to :)

Reply:
I like it, thanks!